valve-monitor/web-app/Config.ts

```diff
@@ -57,7 +58,10 @@ const Config = {
     return Boolean(optionalEnvVar("DEBUG_CONFIG", ""));
   },
   get hubProjectUID() {
-    return requiredEnvVar("HUB_PROJECTUID");
+    return requiredEnvVar("HUB_PROJECT_UID");
+  },
+  get hubFleetUID() {
+    return requiredEnvVar("HUB_FLEET_UID");
```

Comment:
Should this fleet UID be an optional env var? Technically, a device can not be assigned to a fleet, and we can fetch any fleets for the project through the Notehub API.